### PR TITLE
[DONE] Aliaksei Karzhou.Day1 

### PR DESCRIPTION
**Pull request by ghost**
Final release with Vagrant file and 2 scripts (server.sh and agent.sh)

### CODE REVIEW

File: day1/scripts/agent.sh [hunk 1]
```diff
@@ -1,3 +1,13 @@
 #!/bin/bash
+yum install vim -y;
+yum install http://repo.zabbix.com/zabbix/3.2/rhel/7/x86_64/zabbix-release-3.2-1.el7.noarch.rpm -y;
+yum install zabbix-agent -y;
+yum install zabbix-sender -y;
+systemctl start zabbix-agent;
+sed -i '57 s/# DebugLevel=3/DebugLevel=3/' /etc/zabbix/zabbix_agentd.conf
+sed -i '136 s/ServerActive=127.0.0.1/ServerActive=192.168.56.10/' /etc/zabbix/zabbix_agentd.conf
```
Comment (sbeliakou) [inline day1/scripts/agent.sh:8]: omg, do you know how it works?
what will happen on the 2nd run?

File: day1/scripts/agent.sh [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+sed -i '95 s/Server=127.0.0.1/Server=192.168.56.10/' /etc/zabbix/zabbix_agentd.conf
```
Comment (sbeliakou) [inline day1/scripts/agent.sh:9]: the same weird stuff

File: day1/scripts/agent.sh [hunk 3]
```diff
@@ -1,3 +1,13 @@
 #!/bin/bash
+yum install vim -y;
+yum install http://repo.zabbix.com/zabbix/3.2/rhel/7/x86_64/zabbix-release-3.2-1.el7.noarch.rpm -y;
+yum install zabbix-agent -y;
+yum install zabbix-sender -y;
+systemctl start zabbix-agent;
```
Comment (sbeliakou) [inline day1/scripts/agent.sh:6]: why do you start it here and restart it later?

File: day1/scripts/agent.sh (hunk 1 above)
Comment (ghost) [reply day1/scripts/agent.sh:8]: In this statement I change only 57 and 136 lines ( for example # DebugLevel=3 to DebugLevel=3). But I know that it's not a good variant.

File: day1/scripts/agent.sh (hunk 3 above)
Comment (ghost) [reply day1/scripts/agent.sh:6]: Sorry it was my mistake, I make agent.sh without sed at first and then only add this block. Of course I should start service in the end.

**Pull request closed without merging**